src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -80,6 +80,26 @@ public static class PropertyNames
             /// </summary>
             [Obsolete("Subject name strategies should now be configured using the serializer's configuration. In the future, this configuration property will be removed from SchemaRegistryConfig")]
             public const string SchemaRegistryValueSubjectNameStrategy = "schema.registry.value.subject.name.strategy";
+
+            /// <summary>
+            ///     File path to CA certificate(s) for verifying the schema registry's key.
+            /// </summary>
+            public const string SslCaLocation = "schema.registry.ssl.ca.location";
+
+            /// <summary>
+            ///     Ssl keystore location.
```

Comment:
SSL

Reply:
Is this really a Keystore (which is a java thing), or some other format (PKCS#12?)?
The expected format needs to be specified